app/models/topical_event.rb

```diff
@@ -72,7 +72,7 @@ def ordered_featured_documents
         image_src: document.dig("image", "url"),
         image_alt: document.dig("image", "alt_text"),
         heading_text: document["title"],
-        description: document["summary"],
+        description: document["summary"].truncate(160, separator: " "),
```

Comment:
Oooh, this truncate method is a new one for me, love that you can pass the separator as well 🤩

Reply:
I couldn't find an existing use of this approach in this app. What do we do for other document types - like organisations that have this same issue?

Just concerned about introducing an inconsistency.